content/documentation/getting-started/index.adoc

```diff
@@ -367,6 +367,17 @@ namespaces:
     - mycorp_test.*
 ----
 
+==== Namespace Selectors
+
+Kiali supports an optional label selector for namespaces which is used to fetch a subset of the available namespaces. The label selector is defined under the namespaces definition. The example below selects all namespaces that have a label `kiali-enabled: true`.
+
+[source,yaml]
+----
+namespaces:
+  label_selector: kiali-enabled=true
+----
+
```

Comment:
I would want to talk about the reason why we originally added this label_selector - and that is to support soft-multitenancy. Particularly, we need to tell people the value of that label selector should be unique across multiple meshes. My suggested addition is below:

```suggestion

Note that when using multiple service meshes (i.e. multiple control planes) in the same cluster, you will want to set the label selector's value to a value unique to each mesh. This is so each mesh's Kiali instance will only select those namespaces within that mesh. 
For an example of using Kiali in this kind of soft multi-tenancy mode, see the [Maistra](https://github.com/Maistra/istio-operator) project.
This is the reason why this `label_selector` will be defined by default to the value of `kiali.io/member-of: <istio_namespace>` if the `deployment.accessible_namespaces` is set to something other than the "all namespaces" value `['**']`. This allows you to have multiple control planes in the same cluster and have each control plane contain its own Kiali instance.

```

Reply:
Updated the docs